### PR TITLE
adds qctool report scripts

### PR DESCRIPTION
**Pull request by ablwr**
CC @kieranjol (whom I can't tag for some reason) for Windows compatibility concerns.

This doesn't explain the huge bulk of the script, but maybe one day?

Thanks to @dericed the patron saint of everything!

### CODE REVIEW

**Comment by kieranjol** [comment]
Ah great! Yes, an explanation of the signnalstats command could come later. Would have to look at other examples of how to break it up gracefully.

File: index.html [hunk 1]
```diff
@@ -964,6 +964,66 @@
...
+            <dd>This very large lump of commands declares the input file and passes in a request for all potential data signal information for a file with one video and one audio track</dd>
+            <dt>-show_frames</dt><dd>asks for information about each frame and subtitle contained in the input multimedia stream</dd>
+            <dt>-show_versions</dt><dd>asks for information related to program and library versions</dd>
+            <dt>-of xml=x=1:q=1</dt><dd>sets the data export format to XML </dd>
+            <dt>-noprivate</dt><dd>hides any private data that might exist in the file</dd>
+            <dt>| gzip</dt><dd>The | is to "pipe" (or push) the data into a compressed file format</dd>
+            <dt><code>></code></dt><dd>redirects the standard output (the data made by ffprobe about the video) </dd>
+            <dt><i>input_file</i>.qctools.xml.gz</dt><dd>names the zipped data output file, which can be named anything but needs the extension qctools.xml.gz for compatibility issues</dd>
```
Comment (kieranjol) [inline index.html:988]: Should this be output file?

File: index.html [hunk 2]
```diff
@@ -964,6 +964,66 @@
...
```
Comment (kieranjol) [inline index.html:1018]: Again, output_file? Would have to changed in the full command further up too. Aside from that, all looks great !

File: index.html (hunk 1 above)
Comment (retokromer) [inline index.html]: No, I guess it’s meant to have the same name as the input file, but with `.qctools.xml.gz` appended which makes sense to me.

Comment (retokromer) [reply index.html:988]: No, I guess it’s meant to have the same name as the input file, but with `.qctools.xml.gz`  appended which makes sense to me.

**Comment by retokromer** [comment]
Thank you, LGTM

File: index.html (hunk 1 above)
Comment (ablwr) [reply index.html:988]: You are right, it *could* be output file but I did as @retokromer said, wanted to use the same as the input file (just as good practice).

**Comment by ablwr** [comment]
I'll make an issue for expanding on the nuances in the signalstats zone but merge for now so people can see the script (I think most people don't realize this can be done at all)

**Review by kieranjol — COMMENTED** [review]
Ah yes I get you. You're creating a sidecar, so sorry about that.

**Comment by dericed** [comment]
Could you add a cross-reference to the corresponding documentation in the qctools repo: https://github.com/bavc/qctools/blob/master/docs/data_format.md#creating-a-qctools-document.

**Comment by ablwr** [comment]
@dericed do you prefer that URL or http://bavc.github.io/qctools/data_format.html ?

**Comment by retokromer** [comment]
I added @dericed’s but can change, of course: https://github.com/amiaopensource/ffmprovisr/pull/133

**Comment by ablwr** [comment]
thank you Reto!

**Pull request merged**